scripts/mongodb/backup-s3.sh

```diff
@@ -15,21 +15,29 @@ DATESTAMP=$(date '+%Y-%m-%d')
 # Retrieve a list of all databases
 DATABASES=$(mongo $HOST:$PORT/admin -u $USER -p $PASSWORD --eval  "printjson(db.adminCommand('listDatabases'))" | grep name | cut -d '"' -f4 | grep -v local)
 
-# Imports the gpg public key
+# Imports gpg keys
 gpg --import /opt/rh/secrets/gpg_public_key
+gpg --list-keys
 
 # For each database archive data and push directly to S3
 for DATABASE in $DATABASES; do
   TIMESTAMP=$(date '+%H:%M:%S')
+  echo "==> Archiving database \"$DATABASE\""
+  mongodump -h $HOST:$PORT -u $USER -p $PASSWORD -d $DATABASE --archive --gzip --authenticationDatabase $AUTH_DB > /tmp/$DATABASE-$TIMESTAMP.dump.gz
+  echo "==> Enncrypting database archive \"$DATABASE\""
```

Comment:
Slight typo here with two n's in the word Encrypted. Normally wouldn't be concerned with this but as the backup logs may be customer facing it should be fixed.